packages/graphql/src/schema/validation/custom-rules/directives/default.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type {
+    DirectiveNode,
+    ObjectTypeDefinitionNode,
+    FieldDefinitionNode,
+    EnumTypeDefinitionNode,
+    StringValueNode,
+    InterfaceTypeDefinitionNode,
+} from "graphql";
+import { Kind } from "graphql";
+import { assertArgumentHasSameTypeAsField } from "../utils/same-type-argument-as-field";
+import { getInnerTypeName, isArrayType } from "../utils/utils";
+import { DocumentValidationError } from "../utils/document-validation-error";
+import { GRAPHQL_BUILTIN_SCALAR_TYPES, isSpatial, isTemporal } from "../../../../constants";
+
+// TODO: schema-generation: save enums as map
+
+export function verifyDefault(enums: EnumTypeDefinitionNode[]) {
+    return function ({
+        directiveNode,
+        traversedDef,
+    }: {
+        directiveNode: DirectiveNode;
+        traversedDef: ObjectTypeDefinitionNode | InterfaceTypeDefinitionNode | FieldDefinitionNode;
+    }) {
+        if (traversedDef.kind !== Kind.FIELD_DEFINITION) {
+            // delegate
```

Comment:
it means we are returning from this visitor because there is another visitor that throws an error in this case

Reply:
More details are described here: https://graphql.org/graphql-js/language/#visit

Reply:
My intention with pointing this out was more that future readers might not find the comment particularly helpful, rather than my not knowing what delegate meant.